service_storagefacility/src/main/scala/no/uio/musit/microservice/storagefacility/dao/storage/KdReportDao.scala

```diff
@@ -0,0 +1,108 @@
+package no.uio.musit.microservice.storagefacility.dao.storage
+
+import com.google.inject.Inject
+import no.uio.musit.service.MusitResults.{ MusitDbError, MusitResult, MusitSuccess }
+import play.api.db.slick.{ DatabaseConfigProvider, HasDatabaseConfigProvider }
+import play.api.libs.concurrent.Execution.Implicits.defaultContext
+import slick.driver.JdbcProfile
+
+import scala.concurrent.Future
+
+/**
+ * Created by ellenjo on 26.09.16.
+ */
+
+class KdReportDao @Inject() (val dbConfigProvider: DatabaseConfigProvider) extends HasDatabaseConfigProvider[JdbcProfile] {
+
+  import driver.api._
+
+  def getReportTotalArea: Future[MusitResult[Double]] = {
+    db.run(
+      sql"""
+         select SUM(area_to)
```

Comment:
FIXME
Please be consistent in case for SQL keywords vs other things in query statements.
example:
```sql
SELECT SUM(area_to)
FROM musark_storage.storage_node s, musark_storage.room r
WHERE s.storage_node_id = r.storage_node_id
AND r.perimeter_security = 1
```
or
```sql
select sum(area_to)
from musark_storage.storage_node s, musark_storage.room r
where s.storage_node_id = r.storage_node_id
and r.perimeter_security = 1
```